specification/sql/resource-manager/Microsoft.Sql/preview/2020-02-02-preview/DevOpsAudit.json

```diff
@@ -0,0 +1,399 @@
+        ],
+        "responses": {
+          "200": {
+            "description": "Successfully retrieved the server DevOps audit policy.",
+            "schema": {
+              "$ref": "#/definitions/ServerDevOpsAuditingPolicy"
+            }
+          },
+          "default": {
+            "description": "*** Error Responses: ***\n\n * 404 ServerNotInSubscriptionResourceGroup - Specified server does not exist in the specified resource group and subscription.\n\n * 404 SubscriptionDoesNotHaveServer - The requested server was not found"
+          }
+        },
+        "x-ms-examples": {
+          "Get a server's DevOps audit policy": {
+            "$ref": "./examples/ServerDevOpsAuditGet.json"
+          }
+        }
+      },
+      "put": {
+        "tags": [
+          "DevOpsAudit"
+        ],
+        "description": "Creates or updates a server's DevOps audit policy.",
+        "operationId": "ServerDevOpsAuditPolicies_CreateOrUpdate",
+        "parameters": [
+          {
+            "$ref": "#/parameters/ResourceGroupParameter"
+          },
+          {
+            "$ref": "#/parameters/ServerNameParameter"
+          },
+          {
+            "name": "devOpsAuditPolicyName",
+            "in": "path",
+            "description": "The name of the devops audit policy. This should always be 'default'.",
+            "required": true,
+            "type": "string"
+          },
+          {
+            "name": "parameters",
+            "in": "body",
+            "description": "Properties of DevOps audit policy",
+            "required": true,
+            "schema": {
+              "$ref": "#/definitions/ServerDevOpsAuditingPolicy"
+            }
+          },
+          {
+            "$ref": "#/parameters/SubscriptionIdParameter"
+          },
+          {
+            "$ref": "#/parameters/ApiVersionParameter"
+          }
+        ],
+        "responses": {
+          "200": {
+            "description": "Successfully updated the DevOps audit settings.",
+            "schema": {
+              "$ref": "#/definitions/ServerDevOpsAuditingPolicy"
+            }
+          },
+          "default": {
+            "description": "*** Error Responses: ***\n\n * 400 InvalidServerDevOpsAuditPolicyCreateRequest - The create server DevOps audit policy request does not exist or has no properties object.\n\n * 400 DataSecurityInvalidUserSuppliedParameter - An invalid parameter value was provided by the client.\n\n * 400 DevOpsAuditInvalidStorageAccountCredentials - The provided storage account or access key is not valid.\n\n * 404 ServerNotInSubscriptionResourceGroup - Specified server does not exist in the specified resource group and subscription.\n\n * 404 SubscriptionDoesNotHaveServer - The requested server was not found\n\n * 409 ServerDevOpsAuditPolicyInProgress - Set server DevOps audit is already in progress."
+          },
+          "202": {
+            "description": "Updating the audit DevOps settings is in progress."
```

Comment:
For async create/update via PUT, please follow this guidline. https://github.com/Azure/azure-resource-manager-rpc/blob/master/v1.0/Addendum.md#creatingupdating-using-put